star-dash/star-dash/GameEngine/Systems/PowerUps/SpeedBoostPowerUpSystem.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-class SpeedBoostPowerUpSystem: System, EventListener {
```

Comment:
The `setup` can be removed as well

Reply:
The `System` protocol still conforms to `EventListener`, I only removed `EventListener` here since it is unnecessary!